### PR TITLE
feat: remove insurances

### PR DESCRIPTION
**Pull request by myz1237**
Remove all insurance contents

### CODE REVIEW

File: src/api.ts [hunk 1]
```diff
@@ -77,7 +77,6 @@ export interface RoutesRequest {
 export interface RouteOptions {
   integrator?: string // Should contain the identifier of the integrator. Usually, it's dApp/company name.
   fee?: number // 0.03 = take 3% integrator fee (requires verified integrator to be set)
-  insurance?: boolean // Whether the user wants to insure their tx
```
Comment (melianessa) [inline src/api.ts:80]: we can't do this, it will be a breaking change

**Review by melianessa — CHANGES_REQUESTED** [review]

File: src/api.ts (hunk 1 above)
Comment (myz1237) [reply src/api.ts:80]: We will merge it and https://github.com/lifinance/lifi-backend/pull/5026 this one at the same time

Comment (myz1237) [reply src/api.ts:80]: Possilbe progress: merge this types, and publish it, then update my pr on BE, finally merge my be pr, what you think

Comment (myz1237) [reply src/api.ts:80]: fast and safe, I reckon

Comment (melianessa) [reply src/api.ts:80]: I mean not for the BE but for the partners
We can't just remove some property from the request 😁

Comment (myz1237) [reply src/api.ts:80]: ok, let's keep what the jira ticket said, refuse any request with insurance parameter, send an announcement to partners, and move it

**Comment by myz1237** [comment]
Pr gonna be moved to a draft after be completes the phase one

**Comment by chybisov** [comment]
@myz1237 we already have a PR for this #246

**Comment by myz1237** [comment]
duplicated pr

**Pull request closed without merging**